app/main.py

```diff
@@ -1,6 +1,31 @@
+import os
+import requests
+
+API_KEY = os.environ["API_KEY"]
+
+CITY = "Paris"
```

Comment:
You have hardcoded the 'CITY' constant. This could limit the flexibility of your code. Consider getting the city as an environment variable, like you did with the API_KEY. This will allow you to change the city without modifying the code. For example, you can do this: `CITY = os.environ['CITY']`. Don't forget to add the CITY environment variable to your Dockerfile or docker-compose file.